kdl_parser/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.7.2)
 
 project(kdl_parser)
 
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
I don't have a particular problem with it, but I don't remember seeing `CMAKE_CXX_STANDARD_REQUIRED ON` in most of our other packages.  Why have you added it here?

Reply:
Because I copy/pasted it :upside_down_face: 

It seems like `CMAKE_CXX_STANDARD` is optional without it. It probably has no effect on platforms we target since we know C++14 is available.

> [If this property is set to `ON`, then the value of the `CXX_STANDARD` target property is treated as a requirement. If this property is `OFF` or unset, the `CXX_STANDARD` target property is treated as optional and may “decay” to a previous standard if the requested is not available.](https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD_REQUIRED.html#prop_tgt:CXX_STANDARD_REQUIRED)